daisi/src/cpps/logical/algorithms/disposition/round_robin_initiator.h

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 The SOLA authors
+//
+// This file is part of DAISI.
+//
+// DAISI is free software: you can redistribute it and/or modify it under the terms of the GNU
+// General Public License as published by the Free Software Foundation; version 2.
+//
+// DAISI is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+// the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+// Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with DAISI. If not, see
+// <https://www.gnu.org/licenses/>.
+//
+// SPDX-License-Identifier: GPL-2.0-only
+
+#ifndef DAISI_CPPS_LOGICAL_ALGORITHMS_DISPOSITION_ROUND_ROBIN_INITIATOR_H
+#define DAISI_CPPS_LOGICAL_ALGORITHMS_DISPOSITION_ROUND_ROBIN_INITIATOR_H
+
+// #include <deque>
+#include <map>
+#include <queue>
+#include <random>
+
+#include "disposition_initiator.h"
+
+namespace daisi::cpps::logical {
+class RoundRobinInitiator : public DispositionInitiator {
+public:
+  struct ParticipantInfo {
+    uint32_t assignment_ctr = 0;
```

Comment:
I think counter is not such a long word that it you need an abbrevation. Everyone reading the code has to think for a few seconds what ctr could mean.